client/modules/IDE/components/FileNode.jsx

```diff
@@ -153,7 +156,7 @@ export class FileNode extends React.Component {
                   ref={(element) => { this[`fileOptions-${this.props.id}`] = element; }}
                   tabIndex="0"
                   onClick={this.toggleFileOptions}
-                  onBlur={() => setTimeout(this.hideFileOptions, 200)}
+                  //onBlur={() => setTimeout(this.hideFileOptions, 2000)}
```

Comment:
instead of commenting out this code, can you remove it? this is, after all, the point of version control software, right? 😸